src/components/Comments/Comment.js

```diff
@@ -108,7 +109,7 @@ const Comment = ({
 
   const onBlockComponent = (commentParams) => {
     refBlockComponent.current.openBlockComponent({
-      anonimity: false,
+      anonimity: !!commentParams.data?.is_anonymous,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change to set `anonimity` based on `commentParams.data?.is_anonymous` is a good improvement for flexibility and accuracy. It ensures that the anonymity status is correctly reflected based on the comment data.
```diff
-  anonimity: !!commentParams.data?.is_anonymous,
+  anonimity: commentParams.data?.is_anonymous || false,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->